Experimental/conda/environment.yml

```diff
@@ -0,0 +1,11 @@
+name: fab
```

Comment:
I may be misremembering but I think in other places we might have used the name sci-fab or something like that because the name `fab` was already taken on `conda-forge` - I'm just trying to remember _where_ we did this!

Reply:
Found it - it's in the `setup.py`

Reply:
ooh, thank you

Reply:
That's interesting, the name in setup.py doesn't match the github repo name - and we `import fab`, not sci-fab. I'll change the conda environment name to sci-fab but I think you've raised a new issue here - I think perhaps we should change the github repo name too.

Reply:
I couldn't find a `fab` on conda-forge - not sure I searched in the right places?

Reply:
Interestingly, to uninstall fab after installing with pip, i had to type `pip uninstall sci_fab`!